updated indentations and misc

cryptokoala4 opened the pull request:
readme.md updated with proper heading
script.js fixed indentations
index.html fixed indentations
quiz.html moved scripts to before </body>

cryptokoala4 committed:
initial commit

cryptokoala4 committed:
updated README.md and description/website link

cryptokoala4 committed:
removed planning.txt

cryptokoala4 committed:
fixed indentations (initial pull request)

cryptokoala4 committed:
fixed indentations, i hope..

cryptokoala4 committed:
fixed indentations, updated readme.md with proper heading, moved <scripts> to before </body> in quiz.html

cryptokoala4 committed:
added background image, quiz loads question1 from hash, and am not 100% on indentations but should be accurate to my knowledge

cryptokoala4 commented:
quiz.html properly loads a question from questions.js
background image added to html
css fixed to make the entire seem digestable
no promises on indentations - i can't seem to fully grasp the understanding of when an element is the next level. but previous ones should had been fixed.

cryptokoala4 committed:
added progress bar and a total of 10 questions

f3r commented on css/style.css:
```diff
@@ -0,0 +1,30 @@
+}
+
+body {
+  background-image: url("../images/got-background.jpg");
```
if you make the screen tall but not wide, your image will shrink to the width of the page and half the page will be white. You can solve it by finding a larger size image

![image](https://cloud.githubusercontent.com/assets/970858/10448362/f32dbb7a-71be-11e5-9d84-9c0a90be8ab6.png)

f3r commented on index.html:
```diff
@@ -0,0 +1,31 @@
+<head>
+  <meta charset="UTF-8">
+  <title>MBTI</title>
+    <link href="css/bootstrap.min.css" rel="stylesheet">
```
indentation sir

f3r commented on index.html:
```diff
@@ -0,0 +1,31 @@
+    <h2 class="text-center">Explanation</h2>
+    <article class="text-center">This is a popular way of determining personality types, assigning one of two traits each in four different categories to your personality: <br>Introversion (preferring solitude and small groups of people) or Extroversion (preferring socializing)<br>Intuition (gathering information through the mental and abstract) or Sensing (gathering information through the concr ...
+    </article>
+        <h2 class="text-center">Instructions</h2>
```
fix indentation in the whole file pls

cryptokoala4 replied:
planning to rewrite the entire index.html "text". the text there at the moment is copy pasted. low priority, want to do as much javascript as i can in class

f3r commented on index.html:
```diff
@@ -0,0 +1,31 @@
+    To see which Game of Thrones character you mostly match up with, try this game, a totally inaccurate variant on the Myers-Briggs style personality test.
+    </section>
+    <h2 class="text-center">Explanation</h2>
+    <article class="text-center">This is a popular way of determining personality types, assigning one of two traits each in four different categories to your personality: <br>Introversion (preferring solitude and small groups of people) or Extroversion (preferring socializing)<br>Intuition (gathering information through the mental and abstract) or Sensing (gathering information through the concr ...
```
don't use `<br/>` to separate the content into multiple lines, use `<p>`

cryptokoala4 replied:
understood. will do it later because i plan to rewrite this entire block of text.

f3r commented on js/questions.js:
```diff
@@ -0,0 +1,243 @@
+}
+
+var fullQuestions = {
+  q1: {
```
To make it shorter and more readable, can you indent like this?

```js
  q1: {
    question: "You are almost never late for your appointments",
    option1: { type: "feeling",  point: 2 },
    option2: { type: "feeling",  point: 1 },
    option3: { type: "none",     point: 0 },
    option4: { type: "thinking", point: 1 },
    option5: { type: "thinking", point: 2 }
  },
```

f3r commented on js/script.js:
```diff
@@ -0,0 +1,35 @@
+
+  var goQuiz = function(){
+    $('#start-quiz').click(function(){
+    window.location = "./quiz.html";
```
Do you really need to do this when the file loads?

cryptokoala4 replied:
changed 'window.location' to 'location'. thanks

f3r commented on js/questions.js:
```diff
@@ -0,0 +1,243 @@
+  perceiving: 0,
+}
+
+var fullQuestions = {
```
It makes more sense, since you are referencing the questions with a number anyways, to keep them in an array:

``` js
var questions = [
  {
    question: "You are almost never late for your appointments",
    option1: { type: "feeling",  point: 2 },
    option2: { type: "feeling",  point: 1 },
    option3: { type: "none",     point: 0 },
    option4: { type: "thinking", point: 1 },
    option5: { type: "thinking", point: 2 }
  },
  {
    question: "You like to be engaged in an active and fast-paced job.",
    option1: { type: "feeling",  point: 2 },
    option2: { type: "feeling",  point: 1 },
    option3: { type: "none",     point: 0 },
    option4: { type: "thinking", point: 1 },
    option5: { type: "thinking", point: 2 }
  },
```

cryptokoala4 replied:
awesome! changing this now

cryptokoala4 committed:
partially-updated instructions. see git comment for more details

cryptokoala4 commented:
background image not implemented yet
indentations in index.html not fixed because i will rewrite all the content

updated the format inside questions.js, but did not make it into an array because half my functions relies on q+"num" (as in q1, q2, q3) to change questions. planning to implement this if i have time to change half my code

f3r committed:
refactor javascript

cryptokoala4 committed:
Merge pull request #1 from jackywhlee/f3r

wow

cryptokoala4 committed:
merged refactor, then fixed from broken code to have it running (i.e. progress num, start button)

cryptokoala4 committed:
changed directory folder, just comitting to be safe

cryptokoala4 committed:
added basic addition to the personality types

cryptokoala4 committed:
css overhaul on button and quiz-box

cryptokoala4 committed:
personality result properly prints in plain html text

need to add css and 16 combination types/images to match with result

cryptokoala4 committed:
Added 16 combinations of results

still need to improve the index.html texts such as explanation and
instructions.

cryptokoala4 committed:
added percentage breakdown of your results

cryptokoala4 committed:
styled quiz.html, but still thinking how to stretch background-img when the text crops out

f3r commented on js/questions.js:
```diff
@@ -0,0 +1,518 @@
+  return this.questions[this.currentQuestion++];
+};
+//what's prototype
+Quiz.prototype.answer = function(){
```
remove empty method

cryptokoala4 replied:
done and pushed
